elytron/src/main/resources/org/wildfly/extension/elytron/LocalDescriptions.properties

```diff
@@ -4,9 +4,9 @@ elytron.add=Operation adds the Elytron subsystem
 elytron.remove=Operation removes the Elytron subsystem
 # Attributes
 elytron.default-authentication-context=The default authentication context to be associated with all deployments.
-elytron.initial-providers=Reference to the Providers that should be registered ahead of all existing Providers.
-elytron.final-providers=Reference to the Providers that should be registered after all existing Providers.
-elytron.disallowed-providers=A list of providers that are not allowed, and will be removed from the providers list.
+elytron.initial-providers=Reference to the Providers that should be registered ahead of all existing Providers. When set, affects a whole server globally, regardless it is referenced from other subsystems.
```

Comment:
Sorry I missed this earlier, but I wonder if it should read something more like:
```
When set this globally overrides all other $name.
```
For example `When set this globally overrides all other initial-providers.`

@darranl WDYT? I don't really know much about these settings.

Reply:
This is not about overriding any other configuration options (in standalone.xml meaning) - this mean setting of this affect whole server regardless elytron subsystem is "enabled" by referencing http-authentication-factory from undertow for example...

Reply:
As opposite to elytron resources like http-authentication-factory is, which are ignored when they are not referenced from other subsystem (udertow, ejb3...), this configuration option affect server immediately.

Reply:
Hmm... ...okay. What is meant by the elytron subsystem "enabled"? Is there a way for the subsystem to exist, but not have it enabled?

Reply:
When wildfly documentation talks about elytron "enabling", it mean we start to using it - by setting references to elytron resources in undertow subsystem, ejb subsystem and so on. The elytron subsystem is present for the whole time, but when its resources are not referenced, it is not used by wildfly, so we usually say it is disabled. There is only one exception - attributes mentioned in this PR - which is reason for this warning.